public/script.js

```diff
@@ -0,0 +1,56 @@
+const sourceTextContainer = document.querySelector('.source-text-container')
+
+const spanArray = []
+
+const h1 = document.querySelector('h1')
+
+function populateSourceText(data) {
+  sourceTextContainer.innerHTML = ''
+  spanArray.splice(0, spanArray.length)
+  
+  data.forEach((text) => {
```

Comment:
`text` is not the most semantic name here, "for each text in the data" sounds a bit confusing. perhaps `paragraphOfText`?

Reply:
agreed